config/locales/en.yml

```diff
@@ -3215,6 +3215,7 @@ See the %{link} to find out more about %{sitename}'s features and to start using
   order_information: "Order Information"
   new_payment: "New Payment"
   create_or_update_invoice: "Create or Update Invoice"
+  order_has_change_and_invoice_might_not_be_up_to_date: "The order has changed since the last invoice update. The invoice shown here might not be up-to-date anymore."
```

Comment:
It would be good to scope this under a namespace. Using the lazy-lookup scheme: (based on view filename)

```
spree:
  admin: 
   invoices: 
     index: 
       order_has_change_and_invoice_might_not_be_up_to_date: "The order has changed since the last invoice update. The invoice shown here might not be up-to-date anymore."
```

then the following should work (a dot indicates to use lazy lookup):
```
    =t('.order_has_change_and_invoice_might_not_be_up_to_date')
```

Reply:
Hi @dacook 
I tried that but it didn't work.
Please check and let me know.

Reply:
Hi Mohamed, I took a look and you were almost there. I've added a commit to fix it and another suggestion.